### ISSUE TITLE
failed to decode downloaded font

### ISSUE THREAD

**Issue by duynguyen0511** [raoul2000/yii2-bootswatch-asset#5]
Under Chrome Console, I go lots of "failed to decode downloaded font" warning when I am using theme from Bootswatch. I am not sure it is Bootswatch Theme issue ?

```css
Failed to decode downloaded font: http://localhost/assets/2a8cf5fa/fonts/glyphicons-halflings-regular.woff2
Failed to decode downloaded font: http://localhost/assets/2a8cf5fa/fonts/glyphicons-halflings-regular.woff
Failed to decode downloaded font: http://localhost/assets/2a8cf5fa/fonts/glyphicons-halflings-regular.ttf
```

**Comment by raoul2000** [comment]
Hi, 
I will check that tomorrow and let you know...
ciao

**Comment by raoul2000** [comment]
This is not related to yii2-bootswatch-asset but maybe to https://github.com/thomaspark/bootswatch.
A possible explanation is that since 3.3.5, the files in the *font* folder (`vendor\thomaspark\bootswatch\fonts`) files contain a reference to the actual file.

For example `glyphicons-halflings-regular.ttf` contains : 
```
../bower_components/bootstrap/fonts/glyphicons-halflings-regular.ttf
```
(In the previous version it contained the actual ttf font.)
So, I guess that the navigator is not able to find the file as there is no ../bower_components folder.

ciao

**Comment by duynguyen0511** [comment]
I did composer update which said "nothing to update or install". Any idea ?

**Comment by raoul2000** [comment]
the only solution I can see is to downgrade to previous version of bootswatch

**Comment by duynguyen0511** [comment]
would you please show me how to downgrade ?

**Comment by raoul2000** [comment]
With composer you can uninstall yii2-bootswatch-asset 1.2.4 and re-install yii2-bootswatch-asset 1.2.3.
I am not a composer expert but **composer remove raoul2000/yii2-bootswatch-asset** should work.

more about [composer remove](https://getcomposer.org/doc/03-cli.md#remove)

**Comment by duynguyen0511** [comment]
the issue has been fixed with the latest bootswatch (v3.3.5+2). I removed the 'vendor' directory and run 'composer update' again. Thanks for your help

**Comment by raoul2000** [comment]
you're welcome :smile:

**Comment by raoul2000** [comment]
Following your fix, I have released version 1.2.5 that depends on bootswatch 3.3.5+2
